src/test/cli/rbd/help.t

```diff
@@ -9,107 +9,109 @@ Skip test on FreeBSD as it generates different output there.
-      mirror pool peer remove           Remove a mirroring peer from a pool.
-      mirror pool peer set              Update mirroring peer settings.
-      mirror pool promote               Promote all non-primary images in the
-                                        pool.
-      mirror pool status                Show status for all mirrored images in
-                                        the pool.
-      nbd list (nbd ls)                 List the nbd devices already used.
-      nbd map                           Map image to a nbd device.
-      nbd unmap                         Unmap a nbd device.
-      object-map check                  Verify the object map is correct.
-      object-map rebuild                Rebuild an invalid object map.
-      pool init                         Initialize pool for use by RBD.
-      remove (rm)                       Delete an image.
-      rename (mv)                       Rename image within pool.
-      resize                            Resize (expand or shrink) image.
-      showmapped                        Show the rbd images mapped by the kernel.
-      snap create (snap add)            Create a snapshot.
-      snap limit clear                  Remove snapshot limit.
-      snap limit set                    Limit the number of snapshots.
-      snap list (snap ls)               Dump list of image snapshots.
-      snap protect                      Prevent a snapshot from being deleted.
-      snap purge                        Delete all snapshots.
-      snap remove (snap rm)             Delete a snapshot.
-      snap rename                       Rename a snapshot.
-      snap rollback (snap revert)       Rollback image to snapshot.
-      snap unprotect                    Allow a snapshot to be deleted.
-      status                            Show the status of this image.
-      trash list (trash ls)             List trash images.
-      trash move (trash mv)             Move an image to the trash.
-      trash remove (trash rm)           Remove an image from trash.
-      trash restore                     Restore an image from trash.
-      unmap                             Unmap a rbd device that was used by the
-                                        kernel.
-      watch                             Watch events on image.
+      bench                               Simple benchmark.
+      children                            Display children of snapshot.
+      clone                               Clone a snapshot into a COW child image.
+      copy (cp)                           Copy src image to dest.
+      create                              Create an empty image.
+      diff                                Print extents that differ since a
+                                          previous snap, or image creation.
+      disk-usage (du)                     Show disk usage stats for pool, image
+                                          or snapshot
```

Comment:
need a period(`.`) at the end of the help message

Reply:
(unrelated to this PR)